src/docs/containers/icons.jsx

```diff
@@ -1,20 +1,21 @@
 import _ from 'lodash';
 import React from 'react';
 import { Link } from 'react-router';
+import flow from 'lodash/fp/flow';
 
 import { lucidClassNames } from '../../util/style-helpers';
 import * as components from '../../index';
 
 const { object } = React.PropTypes;
 
 const { Button, Expander } = components;
-const icons = _.chain(components)
-	.reduce((acc, val, key) => (
+const icons = flow(
+	(x) => _.reduce(x, (acc, val, key) => (
 		acc.concat(_.endsWith(key, 'Icon') && key !== 'Icon' ? [[key, val]] : [])
-	), [])
-	.filter(([, Icon]) => !Icon._lucidIsPrivate)
-	.sortBy(([name]) => name)
-	.value();
+	), []),
+	(x) => _.filter(x, ([, Icon]) => !Icon._lucidIsPrivate),
+	(x) => _.sortBy(x, ([name]) => name)
+)(components);
```

Comment:
docs code (code that's not included in the public api) might be a good place for us to try out fp, actually